server/api/db/sqldb/db.py

```diff
@@ -4546,6 +4546,9 @@ def _find_functions(
         if not tag:
             # If no tag is given, we need to outer join to get all functions, even if they don't have tags.
             query = query.outerjoin(Function.Tag, Function.id == Function.Tag.obj_id)
+        elif tag == "*":
+            # Only get functions that have tags
+            query = query.join(Function.Tag, Function.id == Function.Tag.obj_id)
```

Comment:
```suggestion
        else:
            # Only get functions that have tags
            query = query.join(Function.Tag, Function.id == Function.Tag.obj_id)
            if tag != "*":
                query = query.filter(Function.Tag.name == tag)
```
Could have done this way, but doesn't really matter.

Reply:
Done 👍